### DISCUSSION TITLE
循环利用java-SDK插入数据时，出现异常

### DISCUSSION THREAD

**Discussion by LiJie20190102** [milvus-io/milvus#23552] [Q&A and General discussion]
sdk client日志：
![image](https://user-images.githubusercontent.com/53458004/233117686-c4a1db06-4641-48ad-9833-45ac49981855.png)

milvus侧日志：
![image](https://user-images.githubusercontent.com/53458004/233118272-16a17d96-10a9-438a-90e4-c9c7261be261.png)

**注：**每次插入大概2W条数据，然后多线程并发操作多个client来循环插入。
同时我想请问下：
server端返回的“StatusRuntimeException: DEADLINE_EXCEEDED: deadline exceeded after...”这条日志时代表什么意思，我今天
利用客户端将数据load大数据量到内存时，也有"deadline exceeded"字样。详见 #23552

**Comment by xiaofan-luan** [comment]
可能需要收集一下服务端日志提交一下，从你提交的日志来看
1. 写入太快了，compaction做不过来 -> compaction plan skipped due to handler full.
2. 出现了一些找不到segment的情况，可能是符合预期的，原因是大量compaction
3. proxy写入超时了，可能是你的消息队列，kafka/pulsar扛不住了
另外麻烦给一下客户端代码我们可以检查下是否有问题
